package.json

```diff
@@ -97,6 +97,6 @@
   },
   "main": "qunit/qunit.js",
   "engines": {
-    "node": "4.* || 6.* || 7.*"
```

Comment:
I suppose its up to interpretation. Either approach is ultimately fine to me (and unblocks me)